EventFiltering/PWGHF/Macros/train_hf_triggers.py

```diff
@@ -192,10 +192,12 @@ def data_prep(config):
     )
 
     labels_array = np.array([0]*n_bkg + [1]*n_prompt + [2]*n_nonprompt)
-    if test_f < 1:
+    if 0 < test_f < 1:
         train_set, test_set, y_train, y_test = train_test_split(
             df_tot, labels_array, test_size=test_f, random_state=seed_split
         )
+    else:
+        print("ERROR: test_fraction must belong to ]0,1[")
```

Comment:
I would also suggest to exit the execution if this error is raised:
```suggestion
        print("ERROR: test_fraction must belong to ]0,1[")
        sys.exit(0)
```

Reply:
Thanks!